src/Client.php

```diff
@@ -130,4 +131,26 @@ private function mergeAuth(array $options): array
 
         return $options;
     }
+
+    /**
+     * Flatten the query params to string using GuzzleHttp\Psr7\build_query.
+     *
+     * Parse.ly API wants the duplicate aggregator implementation where to
```

Comment:
"the duplicate aggregator implementation" is a little unclear to me.

Otherwise, code looks great! I hope they update their docs too.